doc/6/controllers/security/delete-profile/snippets/delete-profile.js

```diff
@@ -0,0 +1,17 @@
+try {
+  const response = await kuzzle.security.deleteProfile('myProfile');
+  console.log(response);
+  /*
+  { found: true,
```

Comment:
Maybe we can just resolve the promise in case of success ? I dont think anyone is using the response content

Reply:
I agree but as discussed, this is a breaking change. We need to wait for the next major release.

Reply:
A change becomes breaking if the signature changes and people are using it. 
We know this is the case with the security controller.

Reply:
I learned yesterday that this controller was used by one of our client, even though it's undocumented :expressionless: 

So we have no choice but to leave it as it is.